content/download.html

```diff
@@ -101,7 +101,8 @@
     <li><%= generate_download_button("osx", GITHUB_RELEASE_ID, release_tag, sizes) %></li>
     <li><%= generate_download_button("osx", GITHUB_PLAYTEST_ID, playtest_tag, sizes) %></li>
   </ul>
-  <br />
+  <p>OpenRA is also available on <a href="https://caskroom.github.io/">Homebrew Cask</a>:</p>
+  <pre>brew cask install openra</pre>
```

Comment:
Updated. I wasn't sure if it applies here, because it is just a script to install our official Mac ZIPs.